README.md

```diff
@@ -122,7 +122,11 @@ Please use [this guide](CONTRIBUTING.md) to make a contribution to the project a
 
 ### Helm charts release
 
-Open a PR that will:
+New releases are usually made from the default branch. When a bugfix release is
```

Comment:
Can you clarify in thread me this is needed as master has progressed to WIP versions (assuming that's the reason)?